QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -242,6 +249,10 @@ def export_kvstyle_transformed_model_to_onnx(
         for _ in range(n_layer)
     ]
 
+    model_architecture = config.architectures[0]
+    if full_batch_size and model_architecture not in get_lists_of_cb_qeff_models.architectures:
+        raise Exception(f"Continuous batching support for {model_architecture} is not enabled currently")
```

Comment:
raise `NotImplementedError` ?

Reply:
Can we check this at the beginning of `export` function? so user doesn't need to wait till this point?

Reply:
Changed it to NotImplementedError  and moved it under export_for_cloud